matroska_tags.xml

```diff
@@ -68,7 +68,7 @@ Or "Marley Bob" and "Marley Robert Nesta" (no comma needed).</description>
     </tag>
     <tag name="INSTRUMENTS" class="Nested Information" type="UTF-8">
       <description lang="en">The instruments that are being used/played, separated by a comma.
-It **SHOULD** be a child of the following tags: "ARTIST", "LEAD_PERFORMER", or "ACCOMPANIMENT".</description>
+It **MUST** be a child of the another tag, including the following: "ARTIST", "LEAD_PERFORMER", or "ACCOMPANIMENT".</description>
```

Comment:
```suggestion
It **MUST** be a child of another tag, including the following: "ARTIST", "LEAD_PERFORMER", or "ACCOMPANIMENT".</description>
```

Reply:
Applied. (the Github commit system doesn't produce nice commits in the history, nor credit the author of the change)